### PR TITLE
feat(Illustration): Add ProjectReview Illustration 

### PR DESCRIPTION
**Pull request by akshaydmc**
### Overview

Adds the ProjectReview illustration.

### PR Checklist

- [x] Related to designs: https://www.figma.com/design/wdjk01NIOVAuZNleTP2NNZ/%F0%9F%8E%89-AI-Generated-Project-Review?node-id=933-145355&t=RBnZMmqAGU1UjDaN-4
- [x] Related to JIRA ticket: [LABS-254]
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [x] This PR includes testing instructions tests for the code change
- [x] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

Check the new illustration in the Storybook preview: https://6619bc47a254bc105779210a--gamut-preview.netlify.app/?path=/docs/atoms-illustrations--editable-illustration

Check the new illustration in the portal-app preview environment: https://tengu.codecademy.com/learn?page=communityEvents

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/5773)  | [Portal Env](https://tengu.codecademy.com/learn?PR_ENV=portal-app-pr-5773)   |

[LABS-254]: https://skillsoftdev.atlassian.net/browse/LABS-254?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@55.17.4-alpha.d09503.0<br/>@codecademy/gamut-illustrations@0.45.1-alpha.d09503.0<br/>@codecademy/gamut-kit@0.6.401-alpha.d09503.0<br/>@codecademy/styleguide@66.17.5-alpha.d09503.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://66a945a16252b8893a966859--gamut-preview.netlify.app>https://66a945a16252b8893a966859--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/66a945a16252b8893a966859>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]
lgtm!

File: packages/gamut-illustrations/src/index.ts [hunk 1]
```diff
@@ -49,3 +49,4 @@ export * from './Target';
 export * from './TinyBlocks';
 export * from './types';
 export * from './Warning';
+export * from './ProjectReview';
```
Comment (LinKCoding) [inline packages/gamut-illustrations/src/index.ts:52]: small nit: could you put this in alphabetical order? 
after PracticeTarget but before Purpose. Thanks!

Comment (akshaydmc) [reply packages/gamut-illustrations/src/index.ts]: Done!

**Comment by akshaydmc** [comment]
> I also noticed that the svg fits in the top-right-hand corner image
> 
> is that intentional?

It is fixed now

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut@55.17.4-alpha.0651f4.0<br/>@codecademy/gamut-illustrations@0.45.1-alpha.0651f4.0<br/>@codecademy/gamut-kit@0.6.401-alpha.0651f4.0<br/>@codecademy/styleguide@66.17.5-alpha.0651f4.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://66a9e47e29590d799c4c7a36--gamut-preview.netlify.app>https://66a9e47e29590d799c4c7a36--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/66a9e47e29590d799c4c7a36>Deploy Logs</a></p>

**Comment by LinKCoding** [comment]
lgtm! 
add the "Ship It" label and you're good to go!

**Pull request merged**